findLongest.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * This method is to find the longest element in a Array
+ *
+ * @since 5.0.0
+ * @category Array
+ * @param {Array} collection The array to inspect.
+ * @returns {string} Returns  longest element in a Array
+ * @example
+ *
+ * findLongest(["abc", 'abcd', "abcde"])
+ * // => "abcde"
+ */
```

Comment:
no, i am asking about the VS Code extension you use for these comments.

Reply:
no extension, 
open preferences => user Snippets ,
![image](https://user-images.githubusercontent.com/13454418/131237126-e7705dfb-8efb-457a-b999-75c20fd21717.png)

open javascript.json, copy it
```
{
	"HEADER": {

		"prefix": "header",
		"body": [
			"/**",
			"*",
			"* @since 5.0.0",
			"* @categor",
			"* @param",
			"* @returns",
			"* @example",
			"*",
			"*",
			"*",
			"*",
			"*/"
		],
	}
}
```
then reBuild a js file,
input header
![image](https://user-images.githubusercontent.com/13454418/131237181-ad75cce7-44f1-4f9f-8ce5-1ba1b5401a0d.png)
this is all;

Reply:
Thanks